routes/publishedSchedule.js

```diff
@@ -70,19 +78,43 @@ publishedScheduleRouter.get('/season', async (req, res) => {
           PS.notes
         FROM published_schedule PS
         LEFT JOIN catalog C ON PS.event_id = C.id
+        LEFT JOIN day D on PS.day_id = D.id
         WHERE
-          DATE(start_time) >= $1::date AND DATE(start_time) <= $2::date
+          DATE(PS.start_time) >= $1::date AND DATE(PS.start_time) <= $2::date
```

Comment:
i'm pretty sure we're planning on changing the PS.start_time + PS.end_time to type TIME so it can't be converted into a DATE anymore. I think you should switch it to D.event_date